src/applications/check-in/utils/i18n/i18n.js

```diff
@@ -72,7 +72,7 @@ i18n
   });
 
 i18n.on('languageChanged', language => {
-  document.getElementById('content').setAttribute('lang', language);
+  document.getElementById('main')?.setAttribute('lang', language);
```

Comment:
I think it might be worth having a comment here pointing out that we're setting the attribute on `main` because that's where DS components are looking for the language.

Reply:
Agreed, updated with comment.